File: Stock project/src/main/java/com/revature/models/Company.java
package com.revature.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Company {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String companyname;
    private String ticker;
    private double mc;
    private double pricestock;
    private int numstock;
}

File: src/Company.ts
export default class Company {
    id: number;
    companyname: string;
    ticker: string;
    mc: number;
    pricestock: number;
    numstock: number;


    constructor (id: number, companyname: string, ticker: string, mc: number, pricestock: number, numstock: number) {
        this.id = id;
        this.companyname = companyname;
        this.ticker = ticker;
        this.mc = mc;
        this.pricestock = pricestock;
        this.numstock = numstock;
    }
}